prow/gerrit/adapter/adapter.go

```diff
@@ -332,9 +376,18 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 
 	switch change.Status {
 	case client.Merged:
-		// TODO: Do we want to add support for dynamic postsubmits?
-		postsubmits := c.config().PostsubmitsStatic[cloneURI.String()]
-		postsubmits = append(postsubmits, c.config().PostsubmitsStatic[cloneURI.Host+"/"+cloneURI.Path]...)
+		postsubmits := []config.Postsubmit{}
+		if cache != nil {
```

Comment:
Might be easier to just always assume that the cache exists. That is, exit with a fatal error (which you seem to be already doing) if the cache cannot be initialized on startup. This is what the `sub` component does, and it makes the code usage around the cache easier because you don't have to do the "does the cache exist" if/else condition every time.

Reply:
Done